jackson-modules/jackson-conversions-3/src/test/java/com/baeldung/jackson/absentvsnull/JacksonAbsentVsNullUnitTest.java

```diff
@@ -0,0 +1,130 @@
+package com.baeldung.jackson.absentvsnull;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import com.baeldung.jackson.absentvsnull.model.Sample;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.MismatchedInputException;
+
+class JacksonAbsentVsNullUnitTest {
+
+    private static final String ID_FIELD = "id";
+    private static final String PRIMITIVE_FIELD = "amount";
+    private static final ObjectMapper DEFAULT_MAPPER = new ObjectMapper();
+
+    private static final Sample ZERO_ARG_SAMPLE = new Sample();
+    private static final Sample FULL_SAMPLE = new Sample();
```

Comment:
Could you instead add a constructor that takes a set of default values? Or a static factory like `Sample.defaults()`? If you do this, then you wouldn't need a constant here and could simply do `Sample.defaults()` inside each test. This would also remove the need for `@AfterEach`.